### PR TITLE
Add the TiKV tuning guide

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@zhangjinpeng1987 @siddontang @shenli @c4pt0r PTAL.

**Comment by queenypingcap** [comment]
@zhangjinpeng1987 @siddontang @shenli @c4pt0r PTAL.

File: op-guide/tune-TiKV.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+# Use the default value.
+scheduler-concurrency = 102400
+# Generally, use the default value. But the value can be set to be bigger if the write operation are in batches or the row is very big.
+scheduler-worker-pool-size = 4
+```
+## 2.TiKV memory usage
+
+Besides `block cache` and `write buffer`, the system memory is also occupied in the following scenarios:
+
+1). Some of the memory need to be set aside as the system's cache page.
```
Comment (siddontang) [inline op-guide/tune-TiKV.md]: use list 
```
1. 
2.
```

**Comment by siddontang** [comment]
LGTM

**Comment by c4pt0r** [comment]
LGTM

**Pull request merged**